src/DssSpell.sol

```diff
@@ -74,190 +120,324 @@ contract DssSpellAction is DssAction {
+        MKR.transfer(WBC,            20.16 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // Frontier Research - 29.76 MKR - 0xa2d55b89654079987cf3985aeff5a7bd44da15a8
-        MKR.transfer(FRONTIERRESEARCH_WALLET, 29.76 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // PBG         -  9.92 MKR - 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2
+        MKR.transfer(PBG,             9.92 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // GFX Labs - 29.76 MKR - 0x9b68c14e936104e9a7a24c712beecdc220002984
-        MKR.transfer(GFXLABS_WALLET,          29.76 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // Bandhar     -  7.68 MKR - 0xE83B6a503A94a5b764CCF00667689B3a522ABc21
+        MKR.transfer(BANDHAR,         7.68 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // QGov - 29.76 MKR - 0xB0524D8707F76c681901b782372EbeD2d4bA28a6
-        MKR.transfer(QGOV_WALLET,             29.76 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // Libertas    -  7.04 MKR - 0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f
+        MKR.transfer(LIBERTAS,        7.04 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // TRUE NAME - 29.76 MKR - 0x612f7924c367575a0edf21333d96b15f1b345a5d
-        MKR.transfer(TRUENAME_WALLET,         29.76 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // PALC        -  2.24 MKR - 0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A
+        MKR.transfer(PALC,            2.24 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // UPMaker - 9.92 MKR - 0xbb819df169670dc71a16f58f55956fe642cc6bcd
-        MKR.transfer(UPMAKER_WALLET,          9.92 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // Harmony     -  1.92 MKR - 0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2
+        MKR.transfer(HARMONY,         1.92 ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // WBC - 9.92 MKR - 0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47
-        MKR.transfer(WBC_WALLET,              9.92 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // VoteWizard  -  1.6  MKR - 0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1
+        MKR.transfer(VOTEWIZARD,      1.6  ether); // note: ether is a keyword helper, only MKR is transferred here
 
-        // Libertas - 9.92 MKR - 0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f
-        MKR.transfer(LIBERTAS_WALLET,         9.92 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // Navigator   -  0.32 MKR - 0x11406a9CC2e37425F15f920F494A51133ac93072
+        MKR.transfer(NAVIGATOR,       0.32 ether); // note: ether is a keyword helper, only MKR is transferred here
```

Comment:
I see some extra spacing here... I'm not sure if we want to align visually the amounts here. Subjective, but this actually makes it harder to read.

Reply:
Could you please elaborate on why it is harder to read when the numbers are aligned using decimal point? For me it's the most natural way of looking at numbers, but I am also fine to not align the comments (just copy-paste them from the exec sheet, so it's easier to validate) and the `transfer` arguments at all, if both reviewers agree on it. Otherwise, I would keep it as is. @The-Arbiter what's your take?

Reply:
I don't mind if the actual arguments are aligned or not, however the extra spacing in the comments feels weird to read.
In any case, this is not a blocker from my perspective.